test/jdk/sun/security/pkcs11/Signature/SignatureTestPSS.java

```diff
@@ -112,7 +136,14 @@ private static void checkSignature(Provider p, byte[] data, PublicKey pub,
         } catch (InvalidAlgorithmParameterException iape) {
             if (s == PSSUtil.AlgoSupport.MAYBE) {
                 // confirmed to be unsupported; skip the rest of the test
-                System.out.println("    => Skip; no PSS support");
+                System.out.printf("    => Skip; no PSS support public key: %s, private key: %s, " +
+                                  "hash: %s, mgf hash: %s, Algo Support: %s%n",
+                        pub,
+                        priv,
+                        hash,
+                        mgfHash,
+                        s);
+                skipTest = true;
```

Comment:
I think skipTest logic in this test needs further updates. It starts with true as default value, should be false and only set skipTest to true when we encounter a test scenario that is not supported. I think there is also a case where skipTest value is overwritten at line 153 when called in a loop at line 95.